scripts/validationAndPreparation.sh

```diff
@@ -47,7 +48,7 @@ export DATE_OF_NEWEST_CONTRIBUTION=$DATE_OF_NEWEST_CONTRIBUTION
 if [[ !  -z "${NEWEST_CONTRIBUTION}" ]]; then
 	cd /app/
 	echo "starting download; this could take a while..."
-	sftp -i /root/.ssh/id_rsa_worker validationworker@trusted-setup.staging.gnosisdev.com:$NEWEST_CONTRIBUTION /app/.
+	connet_server:$NEWEST_CONTRIBUTION /app/.
 
 	echo "verifying the submission; this could take a while..."
 	if [[ ! -z "${CONSTRAINED}" ]]; then
```

Comment:
Where exactly do we fail when the verification doesn't go through? I don't see `set -e` in this bash script, so wouldn't the script just continue?

Reply:
Added it. Yeah it was definitively necessary